llvm/examples/Kaleidoscope/BuildingAJIT/Chapter1/CMakeLists.txt

```diff
@@ -14,5 +14,3 @@ set(LLVM_LINK_COMPONENTS
 add_kaleidoscope_chapter(BuildingAJIT-Ch1
   toy.cpp
   )
-
-export_executable_symbols(BuildingAJIT-Ch1)
```

Comment:
All tutorial exercises and examples that currently use `export_executable_symbols` will now need `EXPORT_SYMBOLS`. The example/tutorial tests are off by default and Darwin exports symbols by default so we may not see any immediate bot failures, but the tests will fail on Linux where symbols are not exported by default.

Reply:
The expert symbols are pushed to the add_llvm_example by default

Reply:
> The expert symbols are pushed to the add_llvm_example by default

Totally missed that. Thanks for the clarification!